linux-tkg-config/prepare

```diff
@@ -803,10 +803,9 @@ _tkg_srcprep() {
     _disable "SCHED_BMQ"
   elif [ "${_cpusched}" = "cacule" ]; then
     _enable "SCHED_AUTOGROUP" "CACULE_SCHED"
-    _disable "BSD_PROCESS_ACCT" "TASK_XACCT" "CGROUP_CPUACCT" "CGROUP_DEBUG"
+    _disable "BSD_PROCESS_ACCT" "TASK_XACCT" "CGROUP_CPUACCT" "CGROUP_DEBUG" "CACULE_RDB"
     if [ "$_cacule_rdb" = "true" ]; then
       _enable "CACULE_RDB"
-      scripts/config --set-val "RDB_INTERVAL" "$_cacule_rdb_interval"
```

Comment:
This line needs to stay

Reply:
Oh my. How did I manage to make such a mess?